action.yml

```
@@ -47,6 +47,10 @@ inputs:
     required: false
     description: 'Post a comment containing a Native Image build report on pull requests.'
     default: 'false'
+  native-image-merge-pr-reports:
+    required: false
+    description: 'Merge existing comments with new a Native Image build report on pull requests.'
```

If I see this right, the option does not really "merge" reports, it just seems to replace/update PR comments with the latest report. Maybe we should call the option something like `native-image-pr-reports-update` instead?
The description could be adjusted to something like:
```suggestion
    description: 'Instead of posting another comment, update an existing PR comment with the latest Native Image build report.'
```